.env.development

```diff
@@ -5,4 +5,6 @@ DATABASE_USERNAME=username
 DATABASE_PASSWORD=secretpassword
 DATABASE_NAME=dbname
 JWT_SECRET=jwt_awesome_key
-API_PORT=3000
+JWT_SECRET_KEY=jwt_awesome_key
+JWT_EXPIRATION_TIME=3600
+API_PORT=3001
```

Comment:
probably should not be here. I had the docker-compose running in parallel to the direct execution. Different ports could be helpful.

Reply:
Yes, here it works :D